package.json

```diff
@@ -35,7 +35,7 @@
     "style-attr": "^1.0.2",
     "three": "^0.74.0",
     "tween.js": "^15.0.0",
-    "webvr-libs": "^1.0.4"
+    "webvr-libs": "^1.0.7"
```

Comment:
this fixes a rendering issue in `VREffect` in addition to a significant perf issue